boards/views.py

```diff
@@ -33,13 +33,29 @@ class BoardNewView(FormView):
 
 @require_POST
 def create(req):
-    form = CategoryForm(req.POST)
+    form = CategoryForm(req.POST, req.FILES)
+    print("______________")
+    print(req.POST)
     if form.is_valid():
         form.save()
         messages.success(req, "新增成功！")
+    else:
+        print(form.errors)
```

Comment:
```suggestion
        print(form.errors)
```
其他地方有用 `message.error()`，但這裡怎麼用 `print()`？

Reply:
噢不！！這是我debug用的產物，手刀去刪除！